client/rest-high-level/src/main/java/org/elasticsearch/client/RestHighLevelClient.java

```diff
@@ -112,6 +113,14 @@ public boolean ping(Header... headers) throws IOException {
     }
 
     /**
+     * Get the cluster info otherwise provided when calling GET `localhost:9200`
+     */
+    public MainResponse info(Header... headers) throws IOException {
```

Comment:
Is this endpoint documented in the Elasticsearch doc (I didn't find one)? If so, I'd like a link in the Javadoc, otherwise... it's ok :)